### PR TITLE
Add FityIt by @txaiwieser

### PR DESCRIPTION
**Pull request by txaiwieser**
1. [x] Project URL:
2. [x] Update contents.json instead of README 
3. [x] One project per pull request
4. [x] Avoid iOS or open-source in description as it is assumed
5. [x] Use this commit title format if applicable: Add app-name by @github-username
6. [x] Use approved format for your entry

### CODE REVIEW

File: contents.json [hunk 1]
```diff
@@ -16439,6 +16439,33 @@
...
+    },
+    {
+      "title": "FityIt - The shapes game",
+      "description": "FityIt is a challenging game of skill and reflexes. Not only does this game test your reaction time, is super fun and addictive. How long can you survive in FityIt?",
+      "category-ids": [
+         "game",
+         "spritekit"
+      ],
+      "source": "https://github.com/txaiwieser/FityIt",
+      "itunes": "https://itunes.apple.com/us/app/fityit-the-shapes-game/id991444581",
```
Comment (vpeschenkov) [inline contents.json]: Could you remove `us` from the iTunes link?

File: contents.json [hunk 2]
```diff
@@ -16439,6 +16439,33 @@
       ],
       "date_added": "May 26 2018",
       "suggested_by": "@vpeschenkov"
-    }
-  ]
+    },
+    {
+      "title": "FityIt - The shapes game",
```
Comment (vpeschenkov) [inline contents.json]: Could you remove this ` - The shapes game` from the title?

File: contents.json (hunk 1 above)
Comment (txaiwieser) [reply contents.json]: Yes

File: contents.json (hunk 2 above)
Comment (txaiwieser) [reply contents.json]: Ok

**Comment by txaiwieser** [comment]
Done @vpeschenkov ✅

**Pull request merged**